lib/citgm.js

```diff
@@ -32,7 +33,15 @@ exports.windows = windows;
 function find(app, context, next) {
   which(app, function(err, resolved) {
     if (err) {
-      next(Error(app + ' not found in path!'));
+      npmWhich(app, function(err, resolved) {
```

Comment:
we should really only do `which('npm')`` and `which('yarn');` once, instead of here, and for every single `install` and `test`